### PR TITLE
docs(README): clarify usage step

### PR DESCRIPTION
**Pull request by jurdanetac**
Explicitly state to open the Dev Tools and go to the Advanced section, that's where the Preview new design option is

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,8 @@
 > default, which is Mocha.
 
 1. Open the Dark Reader.
-2. Click on `Dev Tools` → `Preview new design` and close the window.
-3. Now click on `See all options` -> `Colors`.
+2. Click on `Dev Tools` → `Advanced` →`Preview new design` and close the window.
+3. Now click on `See all options` → `Colors`.
```
Comment (uncenter) [inline README.md]:
```suggestion
2. Click on **Dev Tools** > **Advanced** > **Preview new design** and close the window.
3. Now click on **See all options** > **Colors**.
```

**Review by moiSentineL — APPROVED** [review]
thanks!

**Review by sgoudham — CHANGES_REQUESTED** [review]
Hey, thanks for the PR!

You'll need to edit the following lines here since the README is generated via [catppuccin/whiskers](https://github.com/catppuccin/whiskers) from this file:
https://github.com/catppuccin/dark-reader/blob/560b302f12d402bd9966d177a023c0659c674a68/dark-reader.tera#L28-L30

**Comment by sgoudham** [comment]
Superceded by #10

**Pull request closed without merging**